docs/schema/clientwebpartenumitems-complextype-apphostwebfeatures.md

```diff
@@ -18,11 +19,9 @@ ms.assetid: 34bab690-a8b8-d080-9dbc-2b8fc1a6342f
 
 ## Type information
 
-|   |   |
-|---|---|
-| **Namespace**  | `http://schemas.microsoft.com/sharepoint/` |
-| **Schema file**  | apphostwebfeatures.xsd |
-| **Extension base**  | None |
+| Namespace | Schema File | Extension Base |
```

Comment:
I like this solution, but the repo guidelines ask that we add headers to tables for these situations, vs creating a new column.  So the ask is for us to **only** change line 22 in this instance.